compiler/noirc_frontend/src/lib.rs

```diff
@@ -19,6 +19,7 @@ pub mod monomorphization;
 pub mod node_interner;
 pub mod parser;
 pub mod resolve_locations;
+pub mod utils;
```

Comment:
```suggestion
pub(crate) mod utils;
```

I don't think we need to make this public.

Reply:
The issue is that those are used in `aztec_macros` and `noirc_evaluator`.